crawl-ref/source/item-use.cc

```diff
@@ -1845,9 +1849,8 @@ bool puton_ring(int slot, bool allow_prompt)
     if (prompt_failed(item_slot))
         return false;
 
-    bool prompt = allow_prompt ? Options.jewellery_prompt : false;
```

Comment:
No need to delete the extra line here (and the one above, before "const bool is_amulet"), as it bloats the diff.